### ISSUE TITLE
@mdx-js/runtime@0.16.8 does not contain 'dist' folder

### ISSUE THREAD

**Issue by benjie** [mdx-js/mdx#386]
### Subject of the issue

@mdx-js/runtime@0.16.8 does not contain 'dist' folder, so it cannot be loaded.

### Steps to reproduce

Look at @mdx-js/runtime@0.16.8 on NPM, e.g.: https://unpkg.com/@mdx-js/runtime@0.16.8/

### Expected behaviour

`require('@mdx-js/runtime')` should not error.

@mdx-js/runtime@0.16.6 on NPM doesn't have this issue, e.g.: https://unpkg.com/@mdx-js/runtime@0.16.6/

### Actual behaviour

`require('@mdx-js/runtime')` produces error due to missing file:

```
{ Error: Cannot find module '@mdx-js/runtime'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18) code: 'MODULE_NOT_FOUND' }
```

**Comment by Jarred-Sumner** [comment]
I'm having the same issue as well.

The default export for `@mdx-js/runtime` on 0.16.6 doesn't work either.

Here's the code I use to make it successfully render MDX at runtime:
```js
import React from "react";
import { transform } from "buble";
import mdx from "@mdx-js/mdx";
import { MDXTag } from "@mdx-js/tag";

export const MDX = ({
  scope = {},
  components = {},
  mdPlugins = [],
  hastPlugins = [],
  children,
  ...props
}) => {
  const fullScope = {
    MDXTag,
    components,
    props,
    ...scope
  };

  const jsx = mdx
    .sync(children, {
      mdPlugins,
      hastPlugins,
      skipExport: true
    })
    .trim();

  const { code } = transform(jsx);

  const keys = Object.keys(fullScope);
  const values = Object.values(fullScope);
  // eslint-disable-next-line no-new-func
  const fn = new Function(
    "_fn",
    "React",
    ...keys,
    `${code}
  return React.createElement(MDXContent, { components, ...props });`
  );

  return fn({}, React, ...values);
};

export default MDX;
```

**Comment by johno** [comment]
This should be fixed in `v0.17.0`. Let us know if you encounter any issues.
